Damage Types Guide

dicedpixels opened the pull request:
Ports [Adding a Damage Type](https://fabricmc.net/wiki/tutorial:damagetypes).

dicedpixels committed:
port damage types guide

dicedpixels committed:
fix image path

dicedpixels committed:
remove the link to the blocks guide

dicedpixels committed:
fix sidebar

modmuss50 committed:
Setup deployment

modmuss50 committed:
Fix publish.yml

modmuss50 committed:
Fix publish

modmuss50 commented on reference/latest/src/main/resources/data/minecraft/tags/damage_type/bypasses_armor.json:
```diff
@@ -0,0 +1,6 @@
+{
```
Everything should ideally be data generated. Even if its not documented.

dicedpixels replied:
Tags can be easily done. Do you think damage_types should be data genned as well?

IMB11 replied:
Yes, I believe you can just use a tag provider that uses the damage type type. Correct me if im wrong!

dicedpixels replied:
Everything is datagenned now. The doc itself will not mention anything about datagen to avoid complexity. Lets add a info box saying this too can be datagenned when the datagen docs are done.

IMB11 committed:
Fix i18n - deadlinks + use Intl instead. (#53)

IMB11 committed:
Locale sidebar URL fixes. (#54)

FabricMCBot committed:
New Crowdin updates (#48)

* New translations index.md (Spanish)

* New translations installing-fabric.md (Spanish)

* New translations installing-mods.md (Spanish)

* New translations updating-fabric.md (Spanish)

* New translations contributing.md (Spanish)

* New translations sidebar_translations.json (Chinese Traditional)

* New translations updating-fabric.md (French)

* New translations index.md (Spanish)

* New translations index.md (Chinese Traditional)

---------

Co-authored-by: Calum H <contact@mineblock11.dev>

IMB11 committed:
Ignore language if index.md is missing. (#55)

Co-authored-by: Fabric Bot <159731069+FabricMCBot@users.noreply.github.com>
Co-authored-by: modmuss <modmuss50@gmail.com>

FabricMCBot committed:
New Crowdin updates (#56)

Benonardo committed:
Small player-facing fixes (#57)

modmuss50 committed:
Replace all "Right Single Quotation Mark"s with an Apostrophe

modmuss50 committed:
More quotation fixes

FabricMCBot committed:
Update Crowdin configuration file

FabricMCBot committed:
New Crowdin updates (#58)

FabricMCBot committed:
Update Crowdin configuration file

Juuxel committed:
index: Fix mroe -> more (#60)

FabricMCBot committed:
New Crowdin updates (#59)

fungoza committed:
small fixes in texts (#62)

FabricMCBot committed:
New Crowdin updates (#64)

* New translations codecs.md (French)

* New translations faq.md (Spanish)

* New translations uploading-logs.md (Spanish)

* New translations windows.md (Russian)

* New translations contributing.md (Chinese Traditional)

* New translations index.md (Chinese Traditional)

* New translations windows.md (Chinese Traditional)

* New translations sidebar_translations.json (Chinese Traditional)

* New translations contributing.md (Portuguese, Brazilian)

* New translations contributing.md (German)

* New translations faq.md (German)

* New translations finding-mods.md (German)

* New translations installing-fabric.md (German)

* New translations installing-mods.md (German)

* New translations crash-reports.md (German)

* New translations uploading-logs.md (German)

* New translations updating-fabric.md (German)

* New translations linux.md (Italian)

* New translations arguments.md (Korean)

* New translations basics.md (Korean)

* New translations suggestions.md (Korean)

* New translations effects.md (Korean)

* New translations potions.md (Korean)

* New translations index.md (Korean)

* New translations sidebar_translations.json (Korean)

* New translations index.md (Ukrainian)

* New translations faq.md (Ukrainian)

* New translations finding-mods.md (Ukrainian)

* New translations index.md (Ukrainian)

* New translations index.md (Vietnamese)

* New translations sidebar_translations.json (Vietnamese)

* New translations contributing.md (Italian)

* New translations codecs.md (Italian)

* New translations index.md (Italian)

* New translations faq.md (Italian)

* New translations finding-mods.md (Italian)

* New translations index.md (Italian)

* New translations installing-fabric.md (Italian)

* New translations windows.md (Italian)

* New translations sidebar_translations.json (Italian)

* New translations contributing.md (Vietnamese)

* New translations index.md (Vietnamese)

* New translations index.md (Vietnamese)

* New translations installing-fabric.md (Vietnamese)

* New translations installing-mods.md (Vietnamese)

* New translations updating-fabric.md (Vietnamese)

* New translations sidebar_translations.json (Vietnamese)

notlin4 committed:
Small Fix (#67)

FabricMCBot committed:
New Crowdin updates (#65)

Superkat32 committed:
Creating Custom Particles Guide Overhaul (#61)

* Creating Custom Particles guide overhaul

All changes are related to the Creating Custom Particles guide.
FEAT: Added things related to the sparkle particle, this includes sentence reworks and code reworks(including file renames)
FEAT: Added an image showcasing the custom particle
FEAT/REFACTOR: Added some tips and information boxes where I felt was fit. Some sections have been replaced to be a tip/info box instead of its own sentence
REFACTOR: Changed the particle's texture from the Fabric logo to a shiny blue sparkle
REFACTOR: Changed the Creating a JSON file and adding textures section to be more easily followed
FIX: JSON is now always properly capitalized when referencing it(not including the actual files)
FIX: Added a period on one of the code comments to be consistent with the rest of the code comments

* Last second image alt text fix

FIX: Somehow my "particle" in "Showcase of the sparkle particle" got lost. I'll also remove "sparkle" just in case that was a style check thing that I didn't notice.

dicedpixels committed:
switch to datagen

dicedpixels committed:
Merge branch 'FabricMC:main' into damage-types

dicedpixels committed:
fix checkstyle errors

IMB11 commented on develop/entities/damage-types.md:
```diff
@@ -0,0 +1,111 @@
+To demonstrate the use of custom damage types, we will use a custom block called _Tater Block_. Let's make is so that
+when a living entity steps on a _Tater Block_, it deals _Tater_ damage.
+
+::: info
```
Remove this callout - block guide doesn't exist yet.

dicedpixels committed:
remove callout for blocks guide

IMB11 commented:
@modmuss50 LGTM
